.github/workflows/release.yml

```
@@ -0,0 +1,53 @@
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - "*.*.*"
+
+name: release
+
+jobs:
+  build:
+    name: Build distributions for PyPI
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+      - name: Set up Python
+        uses: actions/setup-python@61a6322f88396a6271a6ee3565807d608ecaddd1 # v4.7.0
+
+      - name: Install build dependencies
+        run: python -m pip install --upgrade setuptools wheel build
```

```suggestion
        run: python -m pip install --upgrade build
```

Or do the others need to be ugpraded?

---

`wheel` might not be installed so we need to include that; I think I should probably just drop the `--upgrade`.

---

Because `build` does stuff in its own virtualenv, wheel doesn't need to be installed outside.

---

Oh wait, I see what you mean now. Yeah, we only need `build` and no `--upgrade`.